service/src/main/kotlin/fi/espoo/evaka/invoicing/service/FeeDecisionGenerator.kt

```diff
@@ -57,6 +58,10 @@ internal fun Database.Transaction.handleFeeDecisionChanges(
     val prices = getFeeThresholds(from)
     val partnerIds = families.mapNotNull { it.partner }
     val incomes = getIncomesFrom(jsonMapper, incomeTypesProvider, partnerIds + headOfFamily, from)
+    val childIncomes = children.map { child ->
+        child.id to getIncomesFrom(jsonMapper, incomeTypesProvider, listOf(child.id), from)
+    }.toMap()
```

Comment:
This is minor, but the child ids could be included in `val incomes = getIncomesFrom(jsonMapper, incomeTypesProvider, partnerIds + headOfFamily, from)` to minimize the amount of database queries.